vars/buildPlugin.groovy

```diff
@@ -55,6 +78,8 @@ def call(Map addonParams = [:])
 			ThrottleJobProperty.fetchDescriptor().save()
 		}
 
+		if (platform == 'ubuntu-ppa') continue
```

Comment:
Currently this will never be hit as `ubuntu-ppa` isn't listed in `PLATFORMS_VALID`

Reply:
meh, lost in rebase